src/main/connections/MySql/MySqlConnection.ts

```diff
@@ -0,0 +1,50 @@
+import { SqlDefinitionResult, SqlModificationResult, SqlQueryResult } from '@nodescript/adapter-sql-protocol';
+import { FieldPacket, PoolConnection, QueryResult, ResultSetHeader, RowDataPacket } from 'mysql2/promise';
+
+import { SqlError } from '../../global/SqlError.js';
+import { getMySqlTypeByCode } from './MySqlFieldTypeMap.js';
+
+export class MySqlConnection {
+    constructor(
+        protected client: PoolConnection
+    ) {}
+    async define(text: string): Promise<SqlDefinitionResult> {
+        await this.execute(text);
+        return { command: text.split(' ')[0] };
```

Comment:
Wondering if this is at all useful 🤔  Stuff like `CREATE TABLE` and `CREATE INDEX` would just return `CREATE` (also in the original case) — not sure if it's useful for the clients.

If the drivers don't return anything consistently, then I'd probably just drop this field.